src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -131,11 +141,34 @@ class CanvasSaveButton extends PureComponent {
           h: canvas.height,
         };
 
+        let firstCoordinate;
+        let oppositeCoordinate;
+
         if (extent) {
-          const pixelTopLeft = map.getPixelFromCoordinate(getTopLeft(extent));
-          const pixelBottomRight = map.getPixelFromCoordinate(
-            getBottomRight(extent),
-          );
+          firstCoordinate = getTopLeft(extent);
+          oppositeCoordinate = getBottomRight(extent);
+        } else if (coordinates) {
+          // In case of coordinates coming from DragBox interaction:
```

Comment:
This explanation would be nice at the prop description

Reply:
It's comment for developers. A user does not care, he will send an array of coordinates and expects it works.